AnkiHubSoftware/ankihub_addon
# [BUILD-298] Remove the enable subdecks dialog when installing a deck

## Related issues

https://ankihub.atlassian.net/browse/BUILD-298


## Proposed changes

This eliminates the confirmation dialog about subdecks presented to the user when installing a deck. Subdecks are now enabled by default. The user can still disable subdecks from Deck Management, where the same dialog is shown.

## How to reproduce

1. Subscribe and install the AnKing deck.
2. Notice the subdecks confirmation dialog is gone.
3. Go to Deck Management to confirm subdecks are enabled by default.
